lib/toopher_api.rb

```diff
@@ -189,8 +318,25 @@ def parse_request_error(decoded)
   end
 end
 
+# Contains advanced ToopherApi methods associated with Pairings
+class Pairings
+  def initialize(raw)
+    @raw = raw
+  end
+
+  # Check on the status of a previous pairing request
+  #
+  # @param [String] pairing_id The unique string identifier id returned by a previous pairing request.
+  #
+  # @return [Pairing] Information about the pairing request
+  def get_by_id(pairing_id)
+    Pairing.new(@raw.get('pairings/' + pairing_id))
```

Comment:
Any particular reason you use string interpolation for the error messages and not for the urls?